engine/runtime-parser/src/test/java/org/enso/compiler/core/EnsoParserTest.java

```diff
@@ -1432,15 +1433,31 @@ public void ise_184219679() throws IOException {
     """);
   }
 
-  private static void parseTest(String code) throws IOException {
-    parseTest(code, true, true, true);
+  @Test
+  public void testWrongIndentation() throws IOException {
+    var code = """
+    group =
+        space4
+            space8
+          space5
+    """;
+    var ir = parseTest(code);
+    var errors = ir.preorder().filter((node) -> node instanceof Syntax);
+    assertEquals(1, errors.size());
+    var first = (Syntax) errors.head();
+    assertEquals(Syntax.UnexpectedExpression$.MODULE$, first.reason());
```

Comment:
What is the error message?
Is it possible to have the message mention 'invalid indentation level'? That would be much more helpful than just 'unexpected expression'.

Reply:
I guess I need to get that information from the Rust side. How do you want me to do it, @kazcw?